app/src/main/java/com/github/libretube/ui/fragments/CommentsRepliesFragment.kt

```diff
@@ -40,12 +43,16 @@ class CommentsRepliesFragment : Fragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
         val videoId = arguments?.getString(IntentData.videoId) ?: ""
-        val nextPage = arguments?.getString(IntentData.replyPage) ?: ""
+        val comment = Json.decodeFromString(
+            Comment.serializer(),
+            arguments?.getString(IntentData.comment) ?: """{}"""
```

Comment:
Why are you triple quoting `"""{}"""` here?

Reply:
I saw this in official Kotlinx-serialization documentation. Anyway that code is never reached because the comment in the argument is guaranteed.

Reply:
But then we should return instead to make that clear instead of that triple quotes statement

Reply:
I think that might be a multi-line string if I'm not wrong

Reply:
Or instead use `!!`